### ISSUE TITLE
[XAML Controls Gallery(debug)->Collections->ListView] : Sent messages are not visible in HC black & white modes in "List for Messaging or data logging" section              

### ISSUE THREAD

**Issue by v-amnat** [microsoft/Xaml-Controls-Gallery#676]
**Tags**
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS4.3.1, SupportingthePlatform,
 A11ySev2, XAMLControlsGallery, WinUI, Win32, Wipro

**Test Environment**
1. XAML Controls Gallery(debug)  version - 2.0.21.0
2. Win 32/ OS build- 21343.rs_Prerelease.210320-1757

**Steps to reproduce the bug**

1. Launch the XAML Controls Gallery (Debug) App
2. Go to the Collections-> ListView.
3. Hit on Tab key to traverse the "ListView(Carousals)".
4. Observe the "ListView for messaging or Data Logging" section after invoking Send Message button. 

**Actual behavior**
When 'Send Message' button is invoked, the message content displayed is not visible in HighContrast Black and white modes.

**Expected**
All content on the page should remain visible/distinguishable in Windows high contrast modes. Typically, these problems are caused by using icons or images when CSS or text could be used, or by not using semantic code to build interactive components. This is a valuable resource for high contrast mode fixes. 

**User Impact** 
When content or components disappear in high contrast modes or they don’t appear as interactive elements, customers who rely on that feature are blocked from information and functionality of the full user experience within the site/app. 

**MAS Reference**
MAS 4.3.1 - No Disruption of Accessibility Features 

**Attachments**
![Send Message Hc Black mode](https://user-images.githubusercontent.com/74585893/114687958-df491880-9d31-11eb-9590-6b508c4570b1.png)
![Send Message Hc White mode](https://user-images.githubusercontent.com/74585893/114687966-e07a4580-9d31-11eb-8b92-8194381e32b5.png)

**Comment by chigy** [comment]
@anawishnoff fixed this
